mlflow/tracking/utils.py

```diff
@@ -169,12 +162,12 @@ def _is_databricks_uri(uri):
 
 
 def _get_file_store(store_uri):
-    path = urllib.parse.urlparse(store_uri).path
+    path = urllib.parse.urlparse(store_uri).path if store_uri else None
     return FileStore(path)
 
 
 def _is_db_uri(uri):
-    if uri.split(':')[0] not in _DBENGINES:
+    if uri.split(':')[0] not in DB_ENGINES:
```

Comment:
Ah, missed this earlier -- would be great to use `urllib.parse.urlparse(uri).scheme` here too, like for the other `_is_XX_uri`'s

Reply:
yeah -- i should have caught it too. Thanks for flagging this.